cloud/setup/variables.tf

```diff
@@ -0,0 +1,62 @@
+variable "confluent_cloud_api_key" {
+  description = "Confluent Cloud API Key (also referred as Cloud API ID)"
+  type        = string
+  sensitive   = true
+}
+
+variable "confluent_cloud_api_secret" {
+  description = "Confluent Cloud API Secret"
+  type        = string
+  sensitive   = true
+}
+
+variable "confluent_environment_id" {
+  description = "Confluent Environment ID"
+  type        = string
+}
+
+variable "confluent_org_id" {
+  description = "Confluent Organization ID"
+  type        = string
+}
+
+variable "nb_topics" {
+  description = "Number of topics to create"
+  type        = number
+  default     = 100
+}
+
+variable "nb_partitions" {
+  description = "Number of partitions for each topic created"
+  type        = number
+  default     = 6
+}
+
+variable "topic_prefix" {
+  description = "Prefix for topic names. Topic name will be: {topic_prefix}_{i}"
+  type        = string
+  default     = "sample"
+}
+
+variable "k8s_context" {
+  description = "The k8s context to connect to"
+  type        = string
+  default     = "docker-desktop"
```

Comment:
This doesn't work on linux, mostly because I don't have a docker-desktop context.

Could we just leave this variable to "null" by default to pick the default kubectl context?

Reply:
If you're sure having it null will default to `kubectl context` then perfect.
In other cases I would:
* remove the default value
* add "(for example: docker-desktop)" in the `description` field